src/lib/stores/workstreams/types.ts

```diff
@@ -30,6 +30,7 @@ export interface Workstream {
   total: Money;
   title: string;
   desc: string;
+  assignee?: string;
```

Comment:
This probably came from a bad merge?

```suggestion
```

Reply:
No I added that one, did we remove that?